src/Committee/CommitteeFactory.php

```diff
@@ -43,6 +43,7 @@ public function createFromArray(array $data): Committee
             $data['created_at'] ?? 'now'
         );
 
+        $committee->setNameLocked(isset($data['name_locked']) ?: false);
```

Comment:
not a big deal but in this case, if `$data['name_locked'] === false`, then `nameLocked` will still be set to true.

i would recommend the following behavior to avoid any future confusion using this method:
`$committee->setNameLocked(isset($data['name_locked']) ? $data['name_locked'] : false);`

Reply:
Good catch, what about `$data['name_locked'] ?? false`?